.env.dev.sample

```diff
@@ -9,7 +9,7 @@ TOKEN_VERIFICATION_API_URL=https://sign-in-dev.hmpps.service.justice.gov.uk/veri
 API_CLIENT_ID=<user_name>
 API_CLIENT_SECRET=<client_secret>
 SESSION_SECRET=session-secret
-CYPRESS_USERNAME=MAKE_RECALL_DECISION_USER
```

Comment:
How come you've removed these from the sample?

Reply:
because that's not the right user for dev